packages/lodestar/src/api/rest/routes/validator/publishAttestation.ts

```diff
@@ -0,0 +1,48 @@
+import {IFastifyServer} from "../../index";
+import fastify, {DefaultBody, DefaultHeaders, DefaultParams, DefaultQuery} from "fastify";
+import {IApiModules} from "../../../interface";
+import {IncomingMessage, Server, ServerResponse} from "http";
+import {fromRestJson} from "../../utils";
+import {Attestation} from "@chainsafe/eth2.0-types";
+
+interface Body extends DefaultBody {
+  attestation: object;
+}
+
+
+const opts: fastify.RouteShorthandOptions<Server, IncomingMessage, ServerResponse, DefaultQuery, DefaultParams, DefaultHeaders, Body> = {
+  schema: {
+    body: {
+      type: 'object',
+      required: ["attestation"],
+      properties: {
+        attestation: {
+          type: "object"
+        },
+      }
+    },
+  }
+};
+
+export const registerAttestationPublishEndpoint = (fastify: IFastifyServer, modules: IApiModules): void => {
+  fastify.post<DefaultQuery, DefaultParams, DefaultQuery, Body>(
+    '/attestation',
+    opts,
+    async (request, reply) => {
+      try {
+        await modules.opPool.attestations.receive(
+          fromRestJson<Attestation>(
+            request.body.attestation,
+            modules.config.types.Attestation
```

Comment:
The spec says that validators post `IndexedAttestation`, not `Attestation` https://github.com/ethereum/eth2.0-APIs/blob/9daa854389c774335b4c39350b4b2f5faa0aa409/apis/validator/beacon-node-oapi.yaml#L245

do we have to modify accordingly?